pkg/utils/esutil/esutil.go

```diff
@@ -0,0 +1,32 @@
+// TODO: refactor
```

Comment:
What does refactor mean? If you are not satisfied with this PR, should keep refining it.

Reply:
We will refine all events, logging and auditing modules in the next version in order to share elastic-related codes. So far it is a reminder.